docs/decisions/0002-installation-method.md

```diff
@@ -0,0 +1,82 @@
+# How to use template-infra with template applications
+
+* Status: proposed
+* Deciders: [list everyone involved in the decision] <!-- optional -->
+* Date: [YYYY-MM-DD when the decision was last updated] <!-- optional -->
+
+Technical Story: User can follow instructions on how to use template-infra with either of the template applications [#2](https://github.com/navapbc/platform/issues/2)
+
+## Context and Problem Statement
+
+We previously decided to [separate infra and application templates into separate repos](./0001-separate-infra-and-application-templates.md). We need to decide how a user of the platform can use the infra template with one of the application templates or with their own custom application.
+
+## Considered Options
+
+1. User creates a repo from the template-infra repo, and replaces the `app` folder with one copied from an application template or their own.
+2. User copies relevant files from the template-infra repo and from the application template repo.
+
+## Decision Outcome
+
+TBD
+
+## Pros and Cons of the Options
+
+### Create project repo from template-infra repo
+
+Pros
+
+* Fewer things to copy over (github actions, pull request template)
+
+Cons
+
+* Doesn't work with existing repo (e.g. if a user already has an application and wants to add template infra to the repo)
+* May still need to have a setup script or instructions to remove/clean up files that aren't needed (e.g. the example application in template-infra is likely not needed)
+
+### Copy files into existing repo
```

Comment:
This seems the most straightforward. I could also see a scenario where it's just instructions in the readme rather than a script. Is it oversimplifying for the instructions to be something the below?

- Download this repo
- Copy `docs/` and `infra/` into your project repo

Reply:
Yeah I'm leaning towards that which is close to what I had below, except what I have below also includes copying over docker-compose.yml and concatenating the .gitignore files.

Reply:
oh yeah good call i guess that's an oversight on my part. in that case just the docker-compose then

Reply:
also, fyi there's a related proposal https://github.com/navapbc/template-infra/pull/106 for how we handle "installing" the template applications. I think it's a bit more complex than installing the infra, since we need to "hook into" the existing infra.

Reply:
Ah I forgot, I think we also want to copy over the github action workflows for CI and CD once we have those.